src/api/projects/queries/getProject/index.js

```diff
@@ -39,16 +49,28 @@ const resolver = async (
       'tk.id'
     )
     .join('projects as p', 'tk.projectId', 'p.id')
-    .where('p.ownerId', '=', request.user.id)
+    .where('p.id', '=', project.id)
     .count()
     .pluck('count')
     .first()
 
+  console.log(translationKeysCount)
+  console.log(localesCount)
+  console.log(expectedTranslationValuesCount)
+  console.log(actualTranslationValuesCount)
+
   const missingTranslationsCount =
     expectedTranslationValuesCount - actualTranslationValuesCount
-  const completePercentage = Math.round(
-    (actualTranslationValuesCount / expectedTranslationValuesCount) * 100
-  )
+
+  const completePercentage =
+    expectedTranslationValuesCount > 0
+      ? Math.round(
+          (actualTranslationValuesCount / expectedTranslationValuesCount) * 100
+        )
+      : 100
```

Comment:
`expectedTranslationsValuesCount > 0` is redundant since if `translationsKeysCount` is not 0 it will always be above 0, no?